resources/user-guides/attach-data.md

```diff
@@ -107,3 +110,271 @@ After you complete your configuration, verify that your rule looks like the foll
 1. Click **Save**
 2. Rebuild your Launch property and deploy it to the correct Environment.
 
+{% endtab %}
+
+{% tab title="Target" %}
+
+### How to attach additional Target request objects to Target retrieve location events
+
+The following sample shows how to _attach data_ to all outgoing `retrieveLocationContent` Target network requests. To create this type of rule, select your property in Experience Platform Launch and complete the following steps:
+
+1. [Create a new **Rule**](#target-create-rule)
+2. [Select the **Event** you want to trigger the rule](#target-select-an-event)
+3. [Select the **Action** to attach data and define your payload](#target-define-the-action)
+4. [Save and rebuild the property](#target-save-the-rule-and-rebuild-your-property)
+
+### Create a rule<a name="target-create-rule"> </a>
+
+1. On the **Rules** tab, click **Create New Rule**.
+
+{% hint style="info" %}
+If you do not have existing rules for this property, the **Create New Rule** button will be in the middle of the screen. If your property has rules, the button will be in the top right of the screen.
+{% endhint %}
+
+### Select an event<a name="target-select-an-event"> </a>
+
+1. Give your rule an easily recognizable name in your list of rules.
+
+   In this example, the rule is named **Attach additional data to Target retrieve location Events**.
+
+2. Under the **Events** section, click **Add**.
+
+3. From the **Extension** drop-down list, select **Adobe Target**.
+
+4. From the **Event Type** drop-down list, select **Content Requested**.
+
+5. Click **Keep Changes**.
+
+![](../../.gitbook/assets/target-attach-data-event-setup.png)
+
+### Define the action<a name="target-define-the-action"> </a>
+
+1. Under the **Actions** section, click **Add**.
+2. From the **Extension** drop-down list, select **Mobile Core**.
+3. From the **Action Type** drop-down list, select **Attach Data**.
+4. On the right pane, in the **JSON Payload** field, type the data that will be added to this event.
+5. Click **Keep Changes**.
+
+On the right pane, you can add a freeform JSON payload that adds data to an SDK event before an extension that is listening for this event can hear the event. In this example, a custom mbox with custom mbox parameters is added to the event before the Target extension processes it. The added custom mbox will now be added on outgoing Target retrieve location requests.
+
+In the following example, an mbox named **custom_mbox** with an additional mbox parameter named **customparams** is added to the Target event.
```

Comment:
We should mention - In the following example, an additional `TargetRequest` object for the given mbox `custom_mbox` containing the provided Target parameters is added to the Target event.

Reply:
@rymorale, please reword Swarna's suggested text as follows:
"In the following example, an additional TargetRequest object for the given `custom_mbox`, which contains the provided Target parameters, is added to the Target event.